api/src/main/java/org/eclipse/microprofile/openapi/models/servers/Server.java

```diff
@@ -111,19 +111,24 @@ default Server description(String description) {
      * </p> 
      * @return ServerVariables variables (in the future this method will instead return a <code>Map&lt;String, ServerVariable&gt;</code>).
```

Comment:
can we update this line?  Perhaps also mention that changes to the returned Map aren't guaranteed to be reflected in the model?

Reply:
Thank you for catching this, I have updated my commit.